Lux/src/gamma_lut.cpp

```diff
@@ -29,9 +31,9 @@ void printBinary( int n, int i )
     for (k = i - 1; k >= 0; k--) {
  
         if ((n >> k) & 1)
-            std::cout << 1;
+            std :: cout << 1;
```

Comment:
maybe consider
a. using bitset as in https://stackoverflow.com/questions/7349689/how-to-print-using-cout-a-number-in-binary-form
b. accepting ostream& parameter so your binary function can be used in e.g. memory or cerr instead of just cout hardcoded as it is now

Reply:
This is code I copied from https://www.geeksforgeeks.org/program-for-conversion-of-32-bits-single-precision-ieee-754-floating-point-representation/ - I'm not using it at the moment but keeping it around for debugging purposes in case I have trouble with gamma_LUT down the road.

I marked these routines as deprecated and I will remove them when I am satisfied with gamma_LUT's correctness.